packages/scss/src/theming/components/_button.theme.scss

```diff
@@ -7,7 +7,7 @@ $button: (
 	padding: _theme('spacings.smaller') _theme('spacings.standard'),
 	small: (
 		font-size: _theme('sizes.small.font-size'),
-		line-height: _theme('sizes.small.line-height'),
+		line-height: _theme('sizes.smaller.line-height'),
```

Comment:
The small and smaller line-heights used to have the same value. Are we sure that we want them to have different values now?

Reply:
That's what we already do on .smaller : 
<img width="599" alt="image" src="https://user-images.githubusercontent.com/25581936/166246812-767ac487-e0ef-4654-bf67-7c4d688aabb1.png">
But to be honest I'm not even sure the theming should be linked to size variables!

Reply:
Ok, let's use this fix for now. It's a bit dodgy to use different size names so we should think about a more consistent solution for the future.